crates/core/tedge_api/src/health.rs

```diff
@@ -1,25 +1,37 @@
 use crate::mqtt_topics::Channel;
+use crate::mqtt_topics::EntityTopicId;
 use crate::mqtt_topics::MqttSchema;
 use crate::mqtt_topics::ServiceTopicId;
 use clock::Clock;
 use clock::WallClock;
 use log::error;
 use mqtt_channel::MqttMessage;
 use mqtt_channel::Topic;
+use serde::Deserialize;
+use serde::Serialize;
 use serde_json::json;
 use std::process;
 use std::sync::Arc;
 use tedge_utils::timestamp::TimeFormat;
 
-pub const MQTT_BRIDGE_UP_PAYLOAD: &str = "1";
-pub const MQTT_BRIDGE_DOWN_PAYLOAD: &str = "0";
+pub const MOSQUITTO_BRIDGE_PREFIX: &str = "mosquitto-";
+pub const MOSQUITTO_BRIDGE_SUFFIX: &str = "-bridge";
```

Comment:
One needs to avoid `pub const` as this is really a poor way to share knowledge across APIs: the caller needs to know when and how to use these.

```suggestion
const MOSQUITTO_BRIDGE_PREFIX: &str = "mosquitto-";
const MOSQUITTO_BRIDGE_SUFFIX: &str = "-bridge";
```

Here, I would even remove these constants as they are only used by `entity_is_mosquitto_bridge_service()` and give a false sense of flexibility.